.github/workflows/ci-fast-tooling.yml

```diff
@@ -0,0 +1,41 @@
+name: CI - FAST Tooling
+
+on:
+  push:
+    branches: 
+      - release
+  pull_request:
+    branches: 
+      - release 
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Cache multiple paths
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/cache
+          !~/cache/exclude
```

Comment:
It looks like cachefiles are not in this location:

```yml
# npm cache files are stored in `~/.npm` on Linux/macOS
path: ~/.npm
```

per https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows#output-parameters-for-the-cache-action

Reply:
I like this one. it provides caching per environment instead of global. we should use it.